pyracf/common/add_operation_error.py

```diff
@@ -0,0 +1,24 @@
+"""Exception to use when Add operation would alter an existing profile."""
+
+
+class AddOperationError(Exception):
+    """
+    Raised when a profile cannot be added because it already exists.
+    """
+
+    def __init__(self, profile_name: str, class_name: str) -> None:
+        self.message = "Security request made to IRRSMO00 failed."
```

Comment:
Now request is made, so maybe it makes more sense to say:

```python
self.message = "Input validation failed."
```
The same should also be done for `AlterOperationError` as well.

Reply:
Agree, change made in both and corresponding unit testing